### PR TITLE
Improve observing run page ui

### PR DESCRIPTION
**Pull request by jadalilleboe**
Add a little popup to confirm deletion
Hnet com-image
also makes the titles clickable.

fixes fritz-marshal/fritz-beta-feedback#117

### CODE REVIEW

**Comment by mcoughlin** [comment]
Maybe @guynir42 has time to look at this quickly today, conference stuff has me a bit busy.

File: static/js/components/ObservingRunPage.jsx [hunk 1]
```diff
@@ -71,10 +82,64 @@ export function observingRunInfo(observingRun, instrumentList, telescopeList) {
   const result = dt.humanize(true);
 
   return result;
-}
+};
 
-const ObservingRunList = ({ observingRuns, deletePermission }) => {
+const DeleteObservingRunDialog = ({ run, deletePermission }) => {
+  console.log(run);
```
Comment (guynir42) [inline static/js/components/ObservingRunPage.jsx:88]:
```suggestion
```

**Review by guynir42 — APPROVED** [review]
It looks good. Sorry this took so long to review. 

I would be even happier if the button to "show all observing runs" was a little bigger and that the entire text was clickable, not just the little box.

**Comment by jadalilleboe** [comment]
> I would be even happier if the button to "show all observing runs" was a little bigger and that the entire text was clickable, not just the little box.

It is a clickable button now :)

**Comment by guynir42** [comment]
Can we make it a different color so it would be more obvious that it is clickable?

**Comment by jadalilleboe** [comment]
Screen Shot 2022-06-03 at 8 09 17 PM
give it an outline?

**Comment by guynir42** [comment]
Sorry to be so pedantic: but can you also make it blue or something similar to the other buttons?

**Comment by jadalilleboe** [comment]
Maybe the grey would be best (similar to manage scanning profile button on scanning page), blue seems to indicate a submitting/creating action.
Screen Shot 2022-06-04 at 12 41 44 PM

**Comment by guynir42** [comment]
@jadalilleboe this looks perfect, thanks!

**Pull request merged**